src/main/java/org/codehaus/mojo/clirr/ClirrClassFilter.java

```diff
@@ -82,6 +100,22 @@ public boolean isSelected( JavaClass javaClass )
                     result = !SelectorUtils.matchPath( excludes[i], path );
                 }
             }
+
+            if (excludeAnnotated != null && javaClass.getAnnotationEntries() != null)
+            {
+                for (AnnotationEntry annotationEntry : javaClass.getAnnotationEntries())
+                {
+                    final String annotationType = annotationEntry.getAnnotationType();
+                    for ( String annotated: excludeAnnotated )
+                    {
+                        if (annotationType.equals("L" + annotated.replace( '.', '/' ) + ";"))
```

Comment:
It would be better to do this `normalization` of the `excludeAnnotated` items once, in the constructor.

Reply:
@martin-g Done. Please review.

Reply:
Awesome!